account/account.go

```diff
@@ -533,6 +533,27 @@ func (account *Account) WaitForTransactionReceipt(ctx context.Context, transacti
 	}
 }
 
+// AddTransaction calls dynamically one of Add<type>Transaction methods
+//
+// Parameters:
+// - ctx: the context.Context object for the transaction.
+// - txn: this Broadcast Transaction to be called.
+// Returns:
+// - interface{} returns required Broadcast txn return value.
+// - error: an error if any.
+func (account *Account) AddTransaction(ctx context.Context, txn rpc.BroadcastTxn) (interface{}, error) {
```

Comment:
Let's improve the return value. Creating a general "Transaction Response" just like we did with the "Transaction Receipt" might be the solution...

Reply:
Done